Flutter/en/2.4.1.md

```diff
@@ -0,0 +1,3 @@
+# Success Criterion 2.4.1 - Bypass Blocks
+
+See the iOS and Android descriptions of this subject, as it is platform specific.  
```

Comment:
Can you add internal links for iOS and Android, e.g. [iOS](../../ios/2.4.1.md) --> URL might be slightly different